### PR TITLE
ibus-engines.rime: 1.4.0 -> 1.5.0

### PR DESCRIPTION
**Pull request by pmeiyu**
###### Motivation for this change

Update ibus-engines.rime to 1.5.0

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Relase notes)](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 128181` at 41bdfe56 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built successfully:</summary>
  <ul>
    <li>ibus-engines.rime</li>
    <li>rime-data</li>
  </ul>
</details>

File: pkgs/data/misc/rime-data/fetchSchema.nix [hunk 1]
```diff
@@ -0,0 +1,137 @@
+fetchFromGitHub:
+# Generated using https://gist.github.com/pmeiyu/c09a8f9547c9a0cdb0e1f610d41c0138
```
Comment (SuperSandro2000) [inline pkgs/data/misc/rime-data/fetchSchema.nix:2]: Please put this file into nixpkgs.

File: pkgs/data/misc/rime-data/default.nix [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  };
+
+  buildInputs = [ librime ];
+
+  buildFlags = [ "all" ];
+  makeFlags = [ "PREFIX=$(out)" ];
+
+  preBuild = import ./fetchSchema.nix fetchFromGitHub;
+
+  patchPhase = ''
```
Comment (SuperSandro2000) [inline pkgs/data/misc/rime-data/default.nix:21]:
```suggestion
  postPatch = ''
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

**Comment by pmeiyu** [comment]
@SuperSandro2000 Ping.

**Pull request merged**